doc/classes/PackedByteArray.xml

```diff
@@ -352,6 +367,14 @@
 				Reverses the order of the elements in the array.
 			</description>
 		</method>
+		<method name="rfind" qualifiers="const">
+			<return type="int" />
+			<argument index="0" name="value" type="int" />
+			<argument index="1" name="from" type="int" default="-1" />
+			<description>
+				Searches the array in reverse order. Optionally, a start search index can be passed. If negative, the start index is considered relative to the end of the array.
```

Comment:
Also this is not true for the current implementation. For an array with 2 elements -5 (if considered as relative to the end of array) is out of bounds. But with the current implementation it would just search the whole array (and the same would happen for positive values out of bounds).